bodhi/server/models.py

```diff
@@ -2265,15 +2265,15 @@ def new(cls, request, data):
                                f"release value of {up.mandatory_days_in_testing} days"
             })
 
-        if not data.get("from_tag"):
-            log.debug("Setting request for new update.")
-            up.set_request(db, req, request.user.name)
-
         log.debug("Adding new update to the db.")
         db.add(up)
         log.debug("Triggering db flush for new update.")
         db.flush()
```

Comment:
Since Celery workers are on different connections, I think we need to `commit()` the transaction here instead of just flushing. No?

Reply:
yes, update it